cloudinit/config/cc_grub_dpkg.py

```diff
@@ -121,14 +126,16 @@ def handle(name, cfg, _cloud, log, _args):
         return
 
     idevs = util.get_cfg_option_str(mycfg, "grub-pc/install_devices", None)
-    idevs_empty = util.get_cfg_option_str(
-        mycfg, "grub-pc/install_devices_empty", None
+    idevs_empty = util.get_cfg_option_bool(
```

Comment:
Small change here to accommodate deprecating the string option here. Also had to modify the test for this module.